src/backend/TrafficCourts/TrafficCourts.Staff.Service.Test/Controllers/EmailHistoryControllerTest.cs

```diff
@@ -50,30 +50,4 @@ public async void TestGetEmailHistory200Result()
         Assert.Equal(emailHistory1, ((List<EmailHistory>)okResult.Value)[0]);
         Assert.Equal(emailHistory2, ((List<EmailHistory>)okResult.Value)[1]);
     }
-
-    [Fact]
-    public void AllEndpointsShouldImplementAuthorizeAttribute()
```

Comment:
So are there some anonymous endpoints now?

Reply:
I have resolved this issue. The unit test `ControllerAuthorizationTest.controller_actions_require_explict_authorization_attributes` removed the allow anonymous check.